components/chassis.py

```diff
@@ -61,67 +58,75 @@ def __init__(
         self.do_smooth = True
 
         # Create Motor and encoder objects
-        self.steer = phoenix5.WPI_TalonFX(steer_id)
-        self.drive = phoenix5.WPI_TalonFX(drive_id)
+        self.steer = phoenix6.hardware.TalonFX(steer_id)
+        self.drive = phoenix6.hardware.TalonFX(drive_id)
         self.drive_id = drive_id
-        self.encoder = phoenix5.sensors.CANCoder(encoder_id)
+        self.encoder = phoenix6.hardware.CANcoder(encoder_id)
 
         # Reduce CAN status frame rates before configuring
-        self.steer.setStatusFramePeriod(
-            phoenix5.StatusFrameEnhanced.Status_1_General, 250, 10
-        )
-        self.drive.setStatusFramePeriod(
-            phoenix5.StatusFrameEnhanced.Status_1_General, 250, 10
-        )
+        self.steer.get_fault_field().set_update_frequency(4, 0.01)
+        self.drive.get_fault_field().set_update_frequency(4, 0.01)
```

Comment:
1. Are we updating the correct frame(s) we don't care about?
2. Is the frame rate correct?

Reply:
I don't know what is the equivalent of ctre.StatusFrameEnhanced.Status_1_General in pheonix6 but I think the get_fault_field() basically gives general status. 

The frame rate is correct, 4Hz. The previous code was 250 ms per refrash so 4 Hz